src/04-calendar/calls.apib

```diff
@@ -0,0 +1,158 @@
+## Calls [/calls]
+
+### calls.list [POST /calls.list]
+
+Get a list of calls.
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + start_date: `2021-11-21` (string) - Filter on dates that are on or after the given day
+            + end_date: `2022-05-13` (string)  - Filter on dates that are on or before the given day
```

Comment:
Q: On which time property do these filter? Added / scheduled / completed at? We must narrow the naming down to the behaviour there.

Reply:
The description could be made clearer. 
How about: `Filter on dates that are on or before the scheduled day`

Reply:
I think these time based filters must then be named as `scheduled_after` and `scheduled_before`, just like we have this format on other filters with `created_after|before`, on invoices we have`invoice_date_after|before` & ..., for time tracking we have `started_after|before`, `ended_after|before` & ...

The naming convention to these filters is that they must be as close as possible to the info property, so if something is named `scheduled_at` it must be filterable as `scheduled_x`

Reply:
https://github.com/teamleadercrm/core/pull/35891